A28-xds-traffic-splitting-and-routing.md

```diff
@@ -173,12 +158,11 @@ message XdsRouting {
     }
     message Fraction {
```

Comment:
This message isn't needed.  We can just use `google.protobuf.UInt32Value` type for the `match_fraction` field.

Reply:
Done